draft-irtf-cfrg-hpke.md

```diff
@@ -510,10 +510,8 @@ The primary differences from the base case are:
 * The PSK ID is added to the context string used as the `info` input
   to the KDF
 
-This mechanism is not suitable for use with a low-entropy password
-as the PSK.  A malicious recipient that does not possess the PSK can
-use decryption of a plaintext as an oracle for performing offline
-dictionary attacks.
+The PSK SHOULD be at least Nh bytes long, and SHOULD be derived from at least
+Nh bytes of entropy.
```

Comment:
I chose a formulation different from “derived from at least Nh bytes of entropy” in #50 because I think deriving from Nh bytes of entropy does not mean that the result of this derivation keeps those Nh bytes of entropy. (e.g. it could be a hash of an appropriate output length for keeping the entropy but then you cut it in half)

Reply:
Good point! I'm fine with the variant.